web_src/js/features/user-auth-webauthn.js

```diff
@@ -1,61 +1,66 @@
-import $ from 'jquery';
-import {encode, decode} from 'uint8-to-base64';
-import {hideElem, showElem} from '../utils/dom.js';
+import {encodeURLEncodedBase64, decodeURLEncodedBase64} from '../utils.js';
 
 const {appSubUrl, csrfToken} = window.config;
 
-export function initUserAuthWebAuthn() {
-  if ($('.user.signin.webauthn-prompt').length === 0) {
+export async function initUserAuthWebAuthn() {
+  document.getElementById('webauthn-error')?.classList.add('hide');
```

Comment:
`hide`? `gt-hidden` or even `hideELem`?

Reply:
I know, I am just doing review, so I think it's good to catch bugs for a non-WIP PR, especially it has got one approval.

Reply:
I'm surprised this still almost merges cleanly still 😆